src/pages/api/assistant/trip.ts

```diff
@@ -26,13 +26,22 @@ export default handlerWithAssistantClient<TripApiReturnType>({
 
     return tryResult(req, res, async () => {
       return ok(
-        await client.trip({
-          from: tripQuery.from!,
-          to: tripQuery.to!,
-          searchTime: tripQuery.searchTime,
-          transportModes,
-          cursor: tripQuery.cursor!,
-        }),
+        tripQuery.via
+          ? await client.viaTrip({
+              from: tripQuery.from!,
+              to: tripQuery.to!,
+              via: tripQuery.via!,
+              searchTime: tripQuery.searchTime,
+              transportModes,
+              cursor: tripQuery.cursor!,
+            })
+          : await client.trip({
+              from: tripQuery.from!,
+              to: tripQuery.to!,
+              searchTime: tripQuery.searchTime,
+              transportModes,
+              cursor: tripQuery.cursor!,
+            }),
```

Comment:
Is there a reason why we have `viaTrip` and `trip`, but we use the same `singleTrip` function for both via and regular trips? Would it be better if we have the functions `trip` and `singleTrip` and that those functions handled which query to use? Both `viaTrip` and `trip` return `Promise<TripData>`. In my opinion, the `via` parameter is more or less the same as any other optional parameter, such as the cursor.